NYTimes.com.js

```diff
@@ -90,14 +94,46 @@ function scrape(doc, url) {
 			}
 		}
 		item.url = ZU.xpathText(doc, '//link[@rel="canonical"]/@href') || url;
-		var pdfurl = ZU.xpathText(doc, '//div[@id="articleAccess"]//span[@class="downloadPDF"]/a[contains(@href, "/pdf")]/@href');
+		item.libraryCatalog = "NYTimes.com";
+		// Convert title/tags of NYT archives when in all caps: 
+		if (item.title == item.title.toUpperCase())
+				item.title = ZU.capitalizeTitle(item.title, true);
+		var allcaps = false;
+		for (i=0; i < item.tags.length; i++) {
+			if (item.tags[i] == item.tags[i].toUpperCase())
+				allcaps = true;
```

Comment:
why go through this trouble? Just do `item.tags[i] = ZU.capitalizeTitle(item.tags[i], true);` here right away. no need for the allcaps variable and the second loop. (This also works at a tag level whereas your solution tries to fix all tags if one is allcaps -- which could just be an acronym).